README.md

```diff
@@ -0,0 +1,31 @@
+# Tsuki-tag
```

Comment:
FYI there's no need to have that file here as it's unlikely to be ever seen by a user. It's just a nitpick though.